assets/scss/common/_article-listing.scss

```diff
@@ -148,3 +151,39 @@
     font-size: rem(18);
   }
 }
+
+.article-image-holder {
+  background-image: repeating-linear-gradient(0deg, transparent -53%, rgba(157, 187, 190, 0.1) 100%, rgba(236, 241, 243, 1) 0);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  @include small-and-up {
+
+    img {
+      max-height: 340px;
+      width: auto;
```

Comment:
@kirdia if `width: auto;` is applicable to all the screen sizes then we can just define it in parent and leave it out of the all the media queries. whatever you have in parent it remains the same till you override it.